### ISSUE TITLE
Update Readme to mention workaround needed for CRA and wasm.

### ISSUE THREAD

**Issue by campbellbartlett** [ergolabs/ergo-sdk-js#1]
I initially gave his a try by throwing together a quick create-react-app and adding the ergo-sdk-js dependency, without realising there is a workaround required for using web assembly with create-react-app.

It took me a while to work out why, I found the answer by looking at the readme of the ergo-lib-wasm-browser readme.

It would be good to add the same comment about the required workaround to this repositories readme. What do you think?

**Comment by oskin1** [comment]
> I initially gave his a try by throwing together a quick create-react-app and adding the ergo-sdk-js dependency, without realising there is a workaround required for using web assembly with create-react-app.
> 
> It took me a while to work out why, I found the answer by looking at the readme of the ergo-lib-wasm-browser readme.
> 
> It would be good to add the same comment about the required workaround to this repositories readme. What do you think?

@campbellbartlett do you mean this? https://github.com/ergoplatform/sigma-rust/tree/develop/bindings/ergo-lib-wasm#when-i-build-the-ergo-lib-wasm-and-include-the-pkg-folder-as-dependency-in-my-app-i-get-typeerror-textdecoder-is-not-a-constructor

**Comment by campbellbartlett** [comment]
Yeah sort of, actually this one: https://github.com/ergoplatform/sigma-rust/tree/develop/bindings/ergo-lib-wasm#using-with-create-react-app

Is it worth adding a similar note in the readme of this repository?
